modules/minimum-ibm-power-system-requirements.adoc

```diff
@@ -9,7 +9,7 @@
 
 You can install {product-title} version {product-version} on the following {ibm-name} hardware:
 
-* {ibm-power-name}9 or {ibm-power-name}10 processor-based systems
```

Comment:
We can't include Power11 until after GA. Right?

Reply:
Thanks @prb112 for the correction. Updated the PR.